Source/Urho3D/Core/ProcessUtils.cpp

```diff
@@ -412,6 +423,63 @@ ea::string GetPlatformName()
     }
 }
 
+unsigned GetNumPhysicalCPUs()
+{
+    int cpuCount = 0;
+#if defined(IOS)
+    host_basic_info_data_t data;
+    mach_msg_type_number_t infoCount;
+    infoCount = HOST_BASIC_INFO_COUNT;
+    host_info(mach_host_self(), HOST_BASIC_INFO, (host_info_t)&data, &infoCount);
+    cpuCount = data.physical_cpu;
+#elif defined(TVOS)
+    cpuCount = SDL_TVOS_GetActiveProcessorCount();
+#elif defined(APPLE) || defined(__FreeBSD__)
+    size_t size = sizeof(cpuCount);
+    sysctlbyname("hw.physicalcpu", &cpuCount, &size, nullptr, 0);
+#elif defined(__linux__)
+    FILE* fp = fopen("/proc/cpuinfo", "rb");
+    if (fp)
+    {
+        char line[1024];
+        while (fgets(line, sizeof(line), fp))
+        {
+            int id = 0;
+            if (sscanf(line, "core id%*[\t]: %d", &id) == 1)
+                cpuCount = Max(cpuCount, static_cast<unsigned>(id + 1));
+        }
+        fclose(fp);
+    }
+#elif defined(__EMSCRIPTEN__)
+#ifdef __EMSCRIPTEN_PTHREADS__
+    cpuCount = emscripten_num_logical_cores();
+#else
+    cpuCount = 1; // Targeting a single-threaded Emscripten build.
+#endif  // __EMSCRIPTEN_PTHREADS__
+#elif defined(_WIN32)
```

Comment:
Let's use enki::GetNumHardwareThreads for Windows.
Also, we can remove platforms if they add more platforms in their library.

Reply:
`enki::GetNumHardwareThreads` returns number of logical cores, not fit for the purpose

Reply:
Right, I missed it, thanks.
FYI, a bit of context: https://github.com/dougbinks/enkiTS/issues/97

Reply:
Uh oh, given that i made our `GetNumLogicalCPUs()` use that as it is more correct.